_issues/but-she-isnt-trustworthy.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Hillary Clinton’s not trustworthy. She lies all the time, right?"
+categories: deception
+---
+
+Q: Hillary Clinton’s not trustworthy. She lies all the time, right?
+
+A: No. Hillary is actually generally one of the most truthful
+politicians to have been in the presidential races for the past 9
+years!
+
+
+* Politifact.org - an independent fact-checking organization that was
+  winner of the Pulitzer prize in journalism has found that Hillary is
+  second best overall in a field of 20 presidential candidates since
+  2007 (the last 9 years). See
+  [a graph for yourself here](https://datavizblog.com/2016/07/24/political-dataviz-who-lies-more-a-comparison-robert-mann/)
+
+* Specifically for 2016, here is Hillary Clinton vs Donald Trump in
```

Comment:
Rather than saying "here", summarize the content of the link you're providing. Don't force the user to click through. For example, the link might say "Politifact rates x% of Hillary's statements as true, while Trumps true statements are x%." Adjust the verbage as appropriate.